eng/Versions.props

```diff
@@ -192,11 +192,11 @@
     <RoslynMicrosoftVisualStudioExtensionManagerVersion>0.0.4</RoslynMicrosoftVisualStudioExtensionManagerVersion>
     <SourceBrowserVersion>1.0.21</SourceBrowserVersion>
     <SystemBuffersVersion>4.5.1</SystemBuffersVersion>
-    <SystemCompositionVersion>1.0.31</SystemCompositionVersion>
-    <SystemCodeDomVersion>4.7.0</SystemCodeDomVersion>
+    <SystemCompositionVersion>5.0.0</SystemCompositionVersion>
```

Comment:
I think this change requires 17.0 preview3 
i.e. this must be in the VS build for our integration tests https://devdiv.visualstudio.com/DefaultCollection/DevDiv/_git/VS/pullrequest/335165